src/pal/src/configure.cmake

```diff
@@ -167,6 +167,22 @@ check_cxx_symbol_exists(_SC_PHYS_PAGES unistd.h HAVE__SC_PHYS_PAGES)
 check_cxx_symbol_exists(_SC_AVPHYS_PAGES unistd.h HAVE__SC_AVPHYS_PAGES)
 
 check_cxx_source_runs("
+  #include <unistd.h>
```

Comment:
This should be a runtime check. We build our portable dotnet core stuff on RHEL / CentOS 7 where this syscall is not present and we still want it to work on machines that has kernel support for this syscall.